package.json

```diff
@@ -12,6 +12,7 @@
     },
     "dependencies": {
         "@fortawesome/fontawesome-free": "5.14.0",
+        "@linusborg/vue-simple-portal": "^0.1.5",
```

Comment:
what does this package do?

Reply:
A Portal Component for Vuejs, to render DOM outside of a component, anywhere in the document.